langchain/vectorstores/faiss.py

```diff
@@ -314,14 +317,21 @@ def __from(
         docstore = InMemoryDocstore(
             {index_to_id[i]: doc for i, doc in enumerate(documents)}
         )
-        return cls(embedding.embed_query, index, docstore, index_to_id)
+        return cls(
+            embedding.embed_query,
+            index,
+            docstore,
+            index_to_id,
+            normalize_score_fn=normalize_score_fn,
```

Comment:
could also pass in kwargs and drop from fn signature

Reply:
I was on the fence about whether or not to communicate it explicitly.  kwargs works